std/base64.d

```diff
@@ -639,7 +639,7 @@ template Base64Impl(char Map62th, char Map63th, char Padding = '=')
          */
         void popFront()
         {
-            enforce(!empty, new Base64Exception("Cannot call popFront on Encoder with no data remaining"));
+            assert(!empty, new Base64Exception("Cannot call popFront on Encoder with no data remaining"));
```

Comment:
I don't think you can do these.

Reply:
We can get rid of the exception and just leave the string:
```suggestion
            assert(!empty, "Cannot call popFront on Encoder with no data remaining");
```

Reply:
Adjusted.